tests/e2e/go.mod

```diff
@@ -0,0 +1,193 @@
+module e2e
+
+go 1.23.1
+
+toolchain go1.23.2
+
+replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.16
+
+replace github.com/alessio/shellescape => al.essio.dev/pkg/shellescape v1.4.2
```

Comment:
Please `replace` the specific versions that need replacing, to avoid silently downgrading the dependencies later

Reply:
I added them to the require block too, to ensure that the desired versions are used and avoid any potential downgrade issues.

Reply:
Without setting an explicit version to replace, these directives will still replace all future version of the library. For a fake example, say some direct dependency requires `mergo v0.2.0`, but that version has a CVE, so it is replaced. When the direct dependency updates to use e.g. `mergo v0.4.0`, the require block will include v0.4.0, but it will be silently replaced with v0.3.16. So the replacement should be something like
```
replace github.com/imdario/mergo v0.2.0 => github.com/imdario/mergo v0.3.16
```

If the replacements are only needed to address vulnerabilities, you should be able to remove these and run nancy-fixer to re-create them using specific versions being upgraded.